dashboard/app/views/reports/search_for_teachers.html.haml

```diff
@@ -1,3 +1,8 @@
+:css
```

Comment:
this should probably go in [admin.css](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/stylesheets/admin.css.scss)